src/decorator/live-decorator.ts

```diff
@@ -31,6 +33,16 @@ export class KalturaLiveEngineDecorator implements KalturaPlayerTypes.IEngineDec
     }
 
     dispatchEvent(event: any): any {
+        if (event.type === EventTypes.METADATA) {
+            this._plugin.handleTimedMetadata(event);
+        }
+        if (event.type === EventTypes.SEEKING) {
```

Comment:
Change it to `SEEKED` and get rid the timeout

Reply:
awesome hint !! 👍 💯 
updated